files/__addonLocation__/package.json

```diff
@@ -70,17 +70,16 @@
     "@types/ember__template": "^4.0.7",
     "@types/ember__test": "^4.0.6",
     "@types/ember__utils": "^4.0.7",
-    "@typescript-eslint/eslint-plugin": "^7.7.1",
-    "@typescript-eslint/parser": "^7.7.1",<% } %>
+    "@typescript-eslint/eslint-plugin": "^8.13.0",
+    "@typescript-eslint/parser": "^8.13.0",<% } %>
     "@rollup/plugin-babel": "^6.0.4",
     "babel-plugin-ember-template-compilation": "^2.2.5",
     "concurrently": "^9.0.1",
     "ember-source": "^5.4.0",
-    "ember-template-lint": "^6.0.0",<% if (packageManager === 'npm') { %>
```

Comment:
diverging ember-eslint-parser versions causes eslint-plugin-ember to not be able to work with gjs/gts